archivy/models.py

```diff
@@ -66,34 +99,6 @@ def extract_content(self, beautsoup):
 
         return html2text.html2text(str(beautsoup))
 
-    def __init__(self, **kwargs):
-
-        # data has already been processed
-        if kwargs["type"] == "processed-dataobj":
```

Comment:
How will this work with the class method that is called if we are using an already existing dataobj?